common/djangoapps/course_modes/views.py

```diff
@@ -148,6 +150,21 @@ def get(self, request, course_id, error=None):
             "responsive": True,
             "nav_hidden": True,
         }
+
+        if enterprise_enabled():
+            enterprise_learner_data = EnterpriseApiClient().fetch_enterprise_learner_data(user=request.user)['results']
+            if enterprise_learner_data:
+                organizations = organization_api.get_course_organizations(course_id=course.id)
+                context["show_enterprise_context"] = True
```

Comment:
how about passing a dict named `enterprise_context` which would be empty if current user is not associated with any enterprise. otherwise it would contain enterprise data in dictionary.
thoughts ?